recipes/servicex-analysis-utils/recipe.yaml

```diff
@@ -0,0 +1,49 @@
+schema_version: 1
+
+context:
+  name: servicex-analysis-utils
+  version: "1.0.0"
+  python_min: "3.9"
```

Comment:
We don't recommend people to set the min when it is the same as conda-forge so it can get automatically updated when we drop this version. I merged b/c I believe you know what you are doing here.

Reply:
Thanks for merging, @ocefpaf!

Yeah, given that I know that the experimental collaborations that this will be used for are going to expect to have Python 3.9 support after the EOL period we generally care more about matching the upstream Python project Python requirement lower bound than trying to keep pace with the rest of the world's (very reasonable) Python support dropping schedule. So here I have a high expectation that this project will still want to support Python 3.9 after Python 3.10 becomes the new conda-forge global pin of `python_min`.